packages/create-app/templates/default-app/packages/app/src/App.tsx

```diff
@@ -55,7 +48,7 @@ const App = () => (
           />
           <Route
             path="/catalog-import"
-            element={<ImportComponentRouter catalogRouteRef={catalogRouteRef} />}
+            element={<ImportComponentRouter />}
```

Comment:
Yes. It now uses the [`EntityRefLink`](https://github.com/backstage/backstage/blob/2aeb2ee94e8af609492a1aaa163e7cf31b129ae8/plugins/catalog-react/src/components/EntityRefLink/EntityRefLink.tsx) component instead of building the link manually. As soon as that component uses `useRouteRef`, this might be needed again (or not).